File: DCAnalyticsWebShell/ClientApp/src/models/Dependency.ts
import { DataCollectionObectTypes } from "src/enums/enums";
import { Target } from "./Target";

export class Dependency
{
    public OID: number
    public Key: string
    public Target: Target
    public QuestionKey: string
    public TargetObjectKey: string
    public TargetObjectType: DataCollectionObectTypes
    public Deleted: boolean
}

File: DCAnalyticsMobile/DCAnalyticsMobile/Models/Dependency.cs
using DCAnalytics;

namespace DCAnalyticsMobile.Models
{
    public class Dependency: DataCollectionObject
    {
        public DataCollectionObectTypes TargetObjectType { get; set; }
        public Target Target { get; set; }
        public string TargetObjectKey { get; set; }
        public string Template { get; set; }
        public bool Deleted { get; set; }
        public int Occurance { get; set; } = -1;
    }
}